pyproject.toml

```diff
@@ -19,18 +19,17 @@ dynamic = [
   'version',
 ]
 dependencies = [
-  "asdf-standard>=1.0.1",
+  "asdf-standard<2,>=1.0.1",
```

Comment:
These should be exact pins to the latest releases of these packages. `dependabot` should auto increment this via opening PRs.